src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/EditIssuePage.kt

```diff
@@ -49,4 +53,52 @@ class EditIssuePage(
             webElement.sendKeys(text)
         }
     }
+
+}
+
+object PageRteUtil{
+
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    fun clearAndTypeIfPresent(driver: WebDriver, id :String, text: String) {
+        if (driver.isElementPresent(By.id(id))) {
+            val webElement = driver.findElement(By.id(id))
+            driver.wait(timeout = Duration.ofSeconds(2), condition = elementToBeClickable(webElement))
+            webElement.clear()
+            webElement.sendKeys(text)
+        }
+    }
+
+    fun clearAndTypeIfPresentRte(driver: WebDriver,id : String, text: String) {
+        //check richeditor-cover exists in class attribute
+        if (driver.isElementPresent(By.id(id))) {
```

Comment:
This `if` doesn't have an `else`. So it seems that if we pass a wrong `id` or the page is broken, we'll silently ignore that.

Reply:
That seems to be a common practice for all other Page Api.